src/api.ts

```diff
@@ -1,40 +1,39 @@
 import { getPRInfo } from "./queries/pr-query";
 
-// Allow all others to access this, we can 
+// Allow all others to access this, we can
 // tighten this down to the TS URLs if the route is abused
 const headers = {
-  "Content-Type": "text/json",
-  "Access-Control-Allow-Methods": "GET",
-  "Access-Control-Allow-Origin": "*",
-}
+    "Content-Type": "text/json",
+    "Access-Control-Allow-Methods": "GET",
+    "Access-Control-Allow-Origin": "*",
+};
 
 const httpTrigger: import("@azure/functions").AzureFunction = async function (context) {
-    if (!context.req) throw new Error("No request in context")
+    if (!context.req) throw new Error("No request in context");
 
     const notFound = (reason: string) => {
         context.res = {
             headers,
             status: 404,
             body: reason
         };
-    }
+    };
 
     const prNumber = Number(context.req.query.number);
-    if (!prNumber || prNumber === NaN)  return notFound("No PR number")
+    if (!prNumber || isNaN(prNumber))  return notFound("No PR number");
 
-    const info = await getPRInfo(prNumber)
+    const info = await getPRInfo(prNumber);
     const prInfo = info.data.repository?.pullRequest;
 
-    if (!prInfo)  return notFound("No PR metadata")
-    if (!prInfo.comments)  return notFound("No PR comments")
+    if (!prInfo)  return notFound("No PR metadata");
+
+    const welcomeComment = prInfo.comments.nodes!.find(c =>  c && c.author?.login === "typescript-bot" && c.body.includes("<!--typescript_bot_welcome-->"));
```

Comment:
BUG!  Double spaces in `=>  c`!

But more seriously, I think that it's a bit better to use `endsWith` so it does the same as the comment parsing thing -- or even better, just use *that* (the stuff in `util/comment.ts`).